scripts/build/aflgo-build.sh

```diff
@@ -2,7 +2,7 @@
 set -e # exit on error
 
 # Build clang & LLVM
-LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git subversion python2.7 binutils-gold binutils-dev curl wget"
+LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git binutils-gold binutils-dev curl wget python3-distutils"
```

Comment:
Is python3-distutils really needed? On Ubuntu 16.04 and earlier, you need to first add the apt-repo ppa:deadsnakes/ppa before you can install python3-distutils. On my machine, it seemed to build fine without.

Reply:
I've only tried it with Ubuntu 20.04 and 18.04. I didn't check on 16.04, but I remember that there was an issue, if I didn't add `python3-distutils`. And indeed, I retried it and got the following error during the build process of llvm when the package is not installed. 

```
Traceback (most recent call last):
  File "/root/build/llvm_tools/llvm-11.0.0.src/projects/libcxx/utils/merge_archives.py", line 12, in <module>
    import distutils.spawn
ModuleNotFoundError: No module named 'distutils.spawn'
```

it probably works for 16.04 because of an older python version, but that's just a guess. If you tell me that it works without this package on Ubuntu 16.04, I will add a check such that it is only installed when the Ubuntu version is >= 18.04.